ospo-book/content/en/04-chapter.md

```diff
@@ -31,6 +32,19 @@ OSPO day-to-day operations encompass a broad spectrum of activities aimed at enh
 
 - **Giving Advice on Open Source Consumption:** The OSPO considers both the strategic view on which open source projects to consume and on the best practice for using the selected projects. The OSPO should provide reference materials and guidance on how the company should select which open source projects it uses and how it manages them. Guidelines and policy can be purely technical or can include considerations based on open source project health and practices, like the [Secure Supply Chain Consumption Framework (S2C2F)](https://github.com/ossf/s2c2f/blob/main/specification/Secure_Supply_Chain_Consumption_Framework_(S2C2F).pdf).
 
+## Security
+
+Before we discuss daily operations, it's important to provide a few comments about security. In some ways, security is simply a requirement like any other requirement. However, many software developers and their managers received inadequate training and education about security. In addition, security is about countering intelligent adversaries, and it's generally an emergent property (it isn't usually isolated to a single place). Open source software has a *potential* security advantage (it can undergo mass peer review and better meets a security principle called "open design"), however, not all potentials are realized.
+
+Managers of software developers (open or closed source) should have at least some training in how to manage development of *secure* software. Those who haven't may want to take the free [Open Source Security Foundation (OpenSSF)](https://openssf.org) course [*Security for Software Development Managers (LFD125)*](https://training.linuxfoundation.org/training/security-for-software-development-managers-lfd125/).
+
+Anyone who develops software that may deployed in the European Union (EU), or manages those developers, needs to learn the basics of the EU Cyber Resilience Act (CRA). If you have not, you may want to take the free OpenSSF course *Understanding the European Union (EU) Cyber Resilience Act (CRA) (LFEL1001)*.
```

Comment:
The suggestion feature in GitHub doesn't work correctly with a line this long or the combination of bold inside a hyperlink (otherwise I would have offered a one-click suggestion).

Shouldn't it be either:

> Anyone who develops software that **deploys** in the European Union (EU)...

or

> Anyone who develops software that **may be** deployed in the European Union (EU)...

(a hint on treating documentation as code: apply [semantic line breaks](https://sembr.org), they make git diffs, line errors, and code reviews easier while the Markdown still renders the same)

Reply:
You're right "may deployed" should be "may be deployed". I'll fix that right now.

Reply:
Fixed.